boards/nordic/nrf9280pdk/nrf9280pdk_nrf9280_cpuppr_xip.yaml

```
@@ -0,0 +1,14 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+identifier: nrf9280pdk/nrf9280/cpuppr/xip
+name: nRF9280-DK-nRF9280-PPR (MRAM XIP)
+type: mcu
+arch: riscv
+toolchain:
+  - zephyr
+sysbuild: true
+ram: 62
```

This should match the size of `cpuppr_code_data`
```suggestion
ram: 14
```

---

Updated to match the size of the updated `cpuppr_code_data` region.